src/README.md

```diff
@@ -3,180 +3,41 @@
 **Welcome aboard!** 🎉 If this is your first time to our repo, here are a few tips:
 
 - Every folder has a README that explains its purpose.
-- For public docs and specs, start in [docs](../docs).
-- For code and dev docs, start in [src](../src) &nbsp; **← YOU ARE HERE**
-- Read about our [architecture](architecture.md) for context on technologies and structure.
-- Review our [coding guidelines](code.md) before you write/review code.
-- Review the guidance below for how to contribute code.
+- If you want to know how to deploy a FinOps toolkit solution, start with the [documentation](https://aka.ms/finops/toolkit).
+- If you want to know how you can contribute, check out the [contribution guide](../CONTRIBUTING.md).
+- If you want to contribute and need to get started with the code, start in the [wiki](https://github.com/microsoft/finops-toolkit/wiki).
+- If you're looking for code, you're in the right place. &nbsp; **← YOU ARE HERE**
+  <br>...but check the wiki to get on the right branch, understand the folder structure, and learn how to build and deploy.
 
 <br>
 
 On this page:
 
-- [🛫 Get started](#-get-started)
-- [📋 Prerequisites](#-prerequisites)
-- [🗃️ Fork and clone](#️-fork-and-clone)
-- [🌿 Select a branch](#-select-a-branch)
-- [🚀 Deploy](#-deploy)
-- [🔬 Test and verify](#-test-and-verify)
-- [👀 Pull requests](#-pull-requests)
+- [⚡ Quickstart guide](#-quickstart-guide)
 
 ---
 
-## 🛫 Get started
+## ⚡ Quickstart guide
 
-There are many ways to contribute to the FinOps toolkit project, like reporting issues, suggesting features, and submitting or reviewing pull requests. For an overview, refer to the [contribution guide](../CONTRIBUTING.md). This page covers how to contribute to the code.
+1. ⬇️ Install Azure PowerShell and the Bicep CLI. [Visual Studio Code](https://code.visualstudio.com) is recommended.
+2. ▶️ Start in the `dev` branch (or applicable [feature branch](../docs-wiki/branching.md#-important-branches)).
```

Comment:
These links won't work in Wiki mode (as it's not relative). Should rather point to the wiki link to the same file (full URL).

Reply:
Good catch!